client/components/plan-storage/index.jsx

```diff
@@ -0,0 +1,54 @@
+/**
+ * External dependencies
+ */
+import classNames from 'classnames';
+import React from 'react';
+import { connect } from 'react-redux';
+import noop from 'lodash/noop';
+
+/**
+ * Internal dependencies
+ */
+import QueryMediaStorage from 'components/data/query-media-storage';
+import { getMediaStorage } from 'state/sites/media-storage/selectors';
+import PlanStorageButton from 'components/plan-storage/button';
+
+const PlanStorage = React.createClass( {
+
+	displayName: 'PlanStorage',
+
+	propTypes: {
+		mediaStorage: React.PropTypes.object,
+		site: React.PropTypes.object.isRequired,
```

Comment:
Few remarks:

- If `isRequired` is specified, we don't need to guard against empty `site` in `render`
- Should `isRequired` be specified? What if we're loading sites from a fresh `localStorage`?
- Could we get away with providing simpler props such as `siteId` ? If not, we should probably pass `{ pure: false }` as options argument to `connect`, yeah?